python/nav/web/seeddb/page/netbox/forms.py

```diff
@@ -65,6 +81,11 @@ class NetboxModelForm(forms.ModelForm):
         label='Virtual instances',
         help_text='The list of virtual instances inside this master device',
     )
+    profiles = forms.MultipleChoiceField(
+        widget=ProfileSelectWithLinks,
+        required=False,
+        choices=ManagementProfile.objects.values_list('id', 'name'),
```

Comment:
This is a bad form.

This line is what is breaking the test suite for you.  This ensures that `choices` for this field are populated **at import time**, which means the database needs to be configured and running as this module is imported.

You need to populate the choices for this field whenever this form is instantiated, not at import time.

Reply:
It would work locally if you have your codebase configured with a running PostgreSQL server.  If running using tox, it would provide a test environment where the database is *not* configured.

As stated, this is intentional to discover these kinds of problems in the codebase. However, this setup is also one of the things that should be adjusted when untangling tox from pytest in that big-test-system-rewrite-in-the-sky-PR that has lived forever - so that the test suite doesn't depend on tox setting up too much of the environment.